dev-python/xcffib/xcffib-0.2.5.ebuild

```diff
@@ -20,8 +20,9 @@ IUSE="test"
 COMMON_DEPEND="x11-libs/libxcb"
 
 RDEPEND="
-	>=dev-python/cffi-0.8.2:=[${PYTHON_USEDEP}]
-	<dev-python/cffi-1:=[${PYTHON_USEDEP}]
+	$(python_gen_cond_dep '>=dev-python/cffi-0.8.2:=[${PYTHON_USEDEP}]' 'python*')
+	$(python_gen_cond_dep '<dev-python/cffi-1:=[${PYTHON_USEDEP}]' 'python*')
```

Comment:
I know this was screwed up already but this dependency is incorrect.

1. The `<` (more restrictive) atom should come first since we have no guarantee that the resolver won't be FIFO, and in this case it must select the newer version first.
2. The dependencies need slot explicitly specified (`0=`), otherwise if cffi gets slotted it will install two different versions.

That said, since the oldest cffi around here is `0.8.6`, you can simply drop the `>=` part.